contentful.json

```diff
@@ -57,23 +57,23 @@
     "Client Side Forms": "No",
     "Plugin System": "No",
     "Customizable UI": "Yes",
-    "Custom Field Types": "No",
+    "Custom Field Types": "Yes",
```

Comment:
This is I guess another point which is unclear. This aims at the question whether it is possible to add new field types in the system (e.g. a markdown field type) via a custom plugin / implementation. (e.g.: Implement a date field type if there is no date field in the system).

Reply:
Contentful supports a JSON object field type which, along with UI Extensions, would allow users to implement completely new field types that aren't included in the standard list of field types.

Reply:
Sounds good. Change manually merged.